### PR TITLE
Windows

### PR DESCRIPTION
**Pull request by minhnhatnoe**
Fixed some grammar errors and moved Verdict constants to models pkg. Also translated most scripts from .sh to .ps1 to facilitate easier ~~development on~~ compilation for Windows.

### CODE REVIEW

File: models/rejudge.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ func batchScoreJobs(subIDs ...int) []*Job {
 
 // Remove the submission's `score`, `penalty` and `verdict`.
 func resetScore(db db.DBContext, subIDs ...int) error {
-	query, params, err := sqlx.In(`UPDATE submissions SET score = NULL, penalty = NULL, verdict = "..." WHERE id IN (?)`, subIDs)
+	query, params, err := sqlx.In(`UPDATE submissions SET score = NULL, penalty = NULL, verdict = `+VerdictIsInQueue+`WHERE id IN (?)`, subIDs)
```
Comment (natsukagami) [inline models/rejudge.go:23]:
```suggestion
	query, params, err := sqlx.In(`UPDATE submissions SET score = NULL, penalty = NULL, verdict =  ? WHERE id IN (?)`, VerdictIsInQueue, subIDs)
```
is less annoying :D

File: embed/embed_dev.go [hunk 2]
```diff
@@ -16,16 +16,16 @@ var Content fs.FS
...
 
 	embedDir := filepath.Join(wd, "embed")
 	stat, err := os.Stat(embedDir)
 	if err != nil {
-		log.Panicf("cannot stat embed directory: %v", err)
+		log.Panicf("Cannot stat embed directory: %v", err)
 	}
 	if !stat.IsDir() {
-		log.Panicf("embed directory is not a directory: %s", embedDir)
+		log.Panicf("Embed directory is not a directory: %s", embedDir)
```
Comment (natsukagami) [inline embed/embed_dev.go:28]: Internal errors in go generally prefer no capitalization. It's panic here tho so whatever :)

File: models/scoreboard.go [hunk 3]
```diff
@@ -134,25 +135,30 @@ func (s *Scoreboard) JSON() JSONScoreboard {
...
+				return a.SolvedProblems > b.SolvedProblems, false
+			}
+			if a.TotalPenalty != b.TotalPenalty {
+				return a.TotalPenalty < b.TotalPenalty, false
+			}
+			return a.User.ID < b.User.ID, true
 	}
+	// in no case should this path be reached
+	// this just stays here in case someone implements a custom contest type
+	log.Panicf("Contest type %s does not exists or not yet implemented", contestType)
```
Comment (natsukagami) [inline models/scoreboard.go:160]:
```suggestion
	log.Panicf("unexpected contest type %s", contestType)
```
Don't go too excessive with comments

File: server/root_ca.go [hunk 4]
```diff
@@ -9,8 +9,8 @@ import (
 	"github.com/pkg/errors"
 )
 
-// ServeHTTPRootCA starts a HTTP server running on `address`
-// serving the root CA from "/ca". It rejects all other requests.
+// ServeHTTPRootCA starts a HTTP server running on `address` using the .pem file at
+// rootCA, serving the root CA from "/ca". It rejects all other requests.
```
Comment (natsukagami) [inline server/root_ca.go:13]: The server is not "using" any `.pem` files, it's an HTTP server. The main server however does ;)

File: worker/compile.go [hunk 5]
```diff
@@ -2,8 +2,10 @@ package worker
 
 // Compiling anything that's more compicated than single file:
 //
-// - Prepare a "compile_%s.sh" file, with %s being the language (cc, go, rs, java, py2, py3, pas).
-// - Prepare any more files as needed. They will all be put into the CWD of the script.
+// - Prepare a "compile_%s.%ext" file, with %s being the language (cc, go, rs, java, py2, py3, pas) and
+//      TODO: %ext being the shell extension (.sh for linux and .bat/.ps1 for windows).
```
Comment (natsukagami) [inline worker/compile.go:6]: This is going to be a massive pain to users... If only some cross-platform scripting language exists ~~not Python~~.
Not sure if we want to fully support both here.

**Review by natsukagami — CHANGES_REQUESTED** [review]

**Comment by natsukagami** [comment]
Also you're on your own with the Windows scripts testing lol.

File: worker/compile.go (hunk 5 above)
Comment (minhnhatnoe) [reply worker/compile.go:6]: How about we use Makefile?

File: embed/embed_dev.go (hunk 2 above)
Comment (minhnhatnoe) [reply embed/embed_dev.go]: Reverted this.

File: server/root_ca.go (hunk 4 above)
Comment (minhnhatnoe) [reply server/root_ca.go]: switch "using" to "serving"

File: worker/compile.go (hunk 5 above)
Comment (natsukagami) [reply worker/compile.go:6]: Makefiles assume bash either, so I'm not sure how that translates to Windows ;)

Comment (minhnhatnoe) [reply worker/compile.go:6]: Checkout this issue https://github.com/natsukagami/kjudge/issues/95 . I think we can implement rudimentary support for "stage" first.

Comment (natsukagami) [reply worker/compile.go:6]: Let's revert this comment since it's not actually implemented yet?

Comment (minhnhatnoe) [reply worker/compile.go:6]: Yeah. Will do ASAP.

File: worker/compile.go [hunk 6]
```diff
@@ -2,8 +2,8 @@ package worker
 
 // Compiling anything that's more compicated than single file:
 //
-// - Prepare a "compile_%s.sh" file, with %s being the language (cc, go, rs, java, py2, py3, pas).
-// - Prepare any more files as needed. They will all be put into the CWD of the script.
+// - Prepare a "compile_%s.%ext" file, with %s being the language (cc, go, rs, java, py2, py3, pas)
+// - Prepare any more files as needed. They will all be put into the CWD of the script
```
Comment (natsukagami) [inline worker/compile.go:6]: Just reminder that this is not yet implemented.

**Review by natsukagami — APPROVED** [review]

**Pull request merged**